components/pages/explorer/RepoTable.tsx

```diff
@@ -182,7 +182,9 @@ const RepoTable = () => {
 		},
 	];
 
-	useArrangerTheme(getTableConfigs({ apiHost: INTERNAL_API_PROXY.ARRANGER, customExporters, theme }));
+	useArrangerTheme(
+		getTableConfigs({ apiHost: INTERNAL_API_PROXY.ARRANGER, customExporters, theme }),
+	);
```

Comment:
Let's keep this PR exclusively about the link... seems like there's a weird thing going on with Prettier's line length that I want to address separately.

```suggestion
	useArrangerTheme(getTableConfigs({ apiHost: INTERNAL_API_PROXY.ARRANGER, customExporters, theme }));
```